core/data/src/main/java/team/ppac/data/mapper/SampleMapper.kt

```diff
@@ -0,0 +1,10 @@
+package team.ppac.data.mapper
+
+import team.ppac.domain.model.SampleImageModel
+import team.ppac.remote.model.sample.SampleResponse
+
+internal fun List<SampleResponse>.toDomain(): List<SampleImageModel> {
+    return this.map { sampleResponse ->
+        SampleImageModel(sampleResponse.download_url, sampleResponse.author)
+    }
+}
```

Comment:
이부분 저번 PR에 제가 올려드린 리뷰같은데, 조금 수정 제안을 드릴까 합니다~ (사실 그때 호다닥 다느라 제대로 못봤던거같기도 ㅋㅅㅋ)
보통 모델 Mapper를 만들 때 다들 어떤식으로 만드는지도 궁금~!! (cc. @EvergreenTree97 @hj1115hj )

저의 경우 모델 단위의 Mapper를 많이 만드는 편이고, 메서드 네이밍도 toData, toDomain, toUiModel과 같이 개별 모델에 대한 매핑 메서드를 작성하는 편입니당. 우리가 매핑을 하려는 대상은 정확히는 리스트가 아니라 모델이니깐유
그래서 위 Mapper 같은 경우에도 아래와 같이 수정될 수 있을 것 같습니다.
```suggestion
internal fun SampleResponse.toDomain(): SampleImageModel = SampleImageModel(
    ~~
)
```
사소한 리뷰이니만큼 다들 가볍게 의견 고고~

Reply:
저도 매퍼 확장 함수를 만들어 두는 편이고, Mapping 대상이 여러개가 될 수 있어 중복이 발생하기에 toDomain 보다는 toSampleImageModel 이런 형태로 쓰는걸 권장합니다!

Reply:
제 생각에도 중복 감안하면 후자가 나은거 같아요~ 반영하겠습니다!